src/alchemlyb/visualisation/convergence.py

```diff
@@ -2,7 +2,10 @@
 from matplotlib.font_manager import FontProperties as FP
 import numpy as np
 
-def plot_convergence(forward, forward_error, backward, backward_error,
+from ..postprocessors.units import get_unit_converter
+
+def plot_convergence(forward=None, forward_error=None, backward=None,
```

Comment:
this is an API change, definitely needs entry in CHANGELOG and versionchanged 0.6.0